packages/eddsa-ticket-pcd/README.md

```diff
@@ -1,3 +1,192 @@
-# `@pcd/eddsa-ticket-pcd`
+<p align="center">
+    <h1 align="center">
+        @pcd/eddsa-ticket-pcd
+    </h1>
+</p>
 
-A PCD representing a Devconnect ticket, whose relevant values were signed by the issuing Zupass server.
+<p align="center">
+    <a href="https://github.com/proofcarryingdata">
+        <img src="https://img.shields.io/badge/project-PCD-blue.svg?style=flat-square">
+    </a>
+    <a href="https://github.com/proofcarryingdata/zupass/blob/main/packages/eddsa-pcd/LICENSE">
+        <img alt="License" src="https://img.shields.io/badge/license-GPL--3.0-green.svg?style=flat-square">
+    </a>
+    <a href="https://www.npmjs.com/package/@pcd/eddsa-ticket-pcd">
+        <img alt="NPM version" src="https://img.shields.io/npm/v/@pcd/eddsa-ticket-pcd?style=flat-square" />
+    </a>
+    <a href="https://npmjs.org/package/@pcd/eddsa-ticket-pcd">
+        <img alt="Downloads" src="https://img.shields.io/npm/dm/@pcd/eddsa-ticket-pcd.svg?style=flat-square" />
+    </a>
+    <a href="https://docs.pcd.team/modules/_pcd_eddsa_pcd.html">
+        <img alt="Docs" src="https://img.shields.io/badge/docs-typedoc-purple.svg?style=flat-square">
+    </a>
+</p>
+
+| This package defines a PCD designed to verify the authenticity of an event ticket signed using an EdDSA key. |
+| ------------------------------------------------------------------------------------------------------------ |
+
+## 🛠 Install
+
+Install the `@pcd/eddsa-ticket-pcd` package with npm:
+
+```bash
+npm i @pcd/eddsa-ticket-pcd
+```
+
+or yarn:
+
+```bash
+yarn add @pcd/eddsa-ticket-pcd
+```
+
+## 📜 Usage
+
+### Prove
+
+```javascript
+import { newEdDSAPrivateKey } from "@pcd/eddsa-pcd"
+import { ITicketData, prove } from "@pcd/eddsa-ticket-pcd"
+import { ArgumentTypeName } from "@pcd/pcd-types"
+import { v4 as uuid } from "uuid";
+
+// Generate a new EdDSA private key.
+const privateKey = newEdDSAPrivateKey()
+
+// Prepare the event ticket to sign.
+// The following are just dummy values used for testing.
+const ticketData: ITicketData = {
+    attendeeName: "test name",
+    attendeeEmail: "user@test.com",
+    eventName: "event",
+    ticketName: "ticket",
+    checkerEmail: "checker@test.com",
+    ticketId: uuid(),
+    eventId: uuid(),
+    productId: uuid(),
+    timestampConsumed: Date.now(),
```

Comment:
```suggestion
    timestampConsumed: 0,
```

a consumed ticket is one that has been checked in, so probably specifying a value here that isn't 0 may be confusing